app/models/corporate_information_page.rb

```diff
@@ -197,4 +198,18 @@ def unique_organisation_and_page_type
       errors.add(:base, "Another '#{display_type_key.humanize}' page was already published for this organisation")
     end
   end
+
+  def unique_worldwide_organisation_and_page_type
+    duplicate_scope = CorporateInformationPage
+      .joins(:edition_worldwide_organisation)
+      .where("edition_worldwide_organisations.worldwide_organisation_id = ?", worldwide_organisation.id)
+      .where(corporate_information_page_type_id:)
+      .where("state not like 'superseded'")
+    if document_id
+      duplicate_scope = duplicate_scope.where("document_id <> ?", document_id)
+    end
+    if duplicate_scope.exists?
+      errors.add(:base, "Another '#{display_type_key.humanize}' page was already published for this worldwide organisation")
```

Comment:
Is `display_type_key` that number we saw in the terminal? If so, do we have access to anything more human readable?

Reply:
No, it's the humanised version of the type's slug, e.g. `about` -> `About` https://github.com/alphagov/whitehall/pull/8009/files#diff-dbc48f170043a2795cb94507f7abc45777dda2aae5c64b40173858c8c2166344R174